src/lightning/fabric/strategies/fsdp.py

```diff
@@ -866,9 +866,13 @@ def _load_raw_module_state_from_path(path: Path, module: Module, world_size: int
 
 def _load_raw_module_state(state_dict: Dict[str, Any], module: Module, world_size: int, strict: bool = True) -> None:
```

Comment:
Alternatively, we could also just not allow loading into a module when it isn't FSDP-wrapped.